tests/boards/mec172xevb_assy6906/qspi/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Why do we need board specific qspi test?

Reply:
I will remove the tests commits from the PR.